chord_metadata_service/chord/data_types.py

```diff
@@ -53,6 +57,7 @@
     },
     DATA_TYPE_EXPERIMENT_RESULT: {
         "label": "Experiment Results",
+        "queryable": False,
```

Comment:
Just for my understanding, is this one false because there is no search schema for it?